packages/integrations/tailwind/README.md

```diff
@@ -27,6 +27,14 @@ pnpx astro add tailwind
 
 If you run into any hiccups, [feel free to log an issue on our GitHub](https://github.com/withastro/astro/issues) and try the manual installation steps below.
 
+Then simply import your stylesheet :tada:.
```

Comment:
Let's actually move this down to the section covering the `applyBaseStyles` option :+1: 

The integration will actually include base styles by default (adding the `@tailwind` directives automatically), so you'll only want to import your own CSS for tailwind if the base styles are disabled

Reply:
I think we should share that also with an example since that was not super clear to me

Reply:
Like @tony-sull said, this import is not actually required with the default config.

This PR is safe to close, and I took a stab at a more holistic README update here: c05a544a. Would love thoughts if you're still seeing a bug or some behavior doesn't match the README.